packages/modals/src/provider/index.tsx

```diff
@@ -1,19 +1,28 @@
-import { useState, useImperativeHandle, forwardRef } from 'react';
-import PropTypes from 'prop-types';
+import React, {
+  forwardRef,
+  ReactNode,
+  useImperativeHandle,
+  useState,
+} from 'react';
 import clsx from 'clsx';
 import styles from './index.module.scss';
 import { Provider } from './context';
 import { useProviderValue } from './hooks';
 
+type ModalProviderProps = {
+  className?: string,
+} & Partial<{
+  [key: string]: unknown;
+}>;
 
-const ModalProvider = forwardRef(({
+const ModalProvider: React.FC = forwardRef<unknown, ModalProviderProps>(({
```

Comment:
wonder why we have to specify `React.FC` here when we never do it anywhere else

Reply:
Definitely shouldn't be `React.FC`. My bad.